### ISSUE TITLE
0Net does not delete deleted optional files from site directory

### ISSUE THREAD

**Issue by mx5kevin** [HelloZeroNet/ZeroNet#2407]
### Step 1: Please describe your environment

  * ZeroNet version: 0.7.1 (Rev:4399)
  * Operating system: Windows 10
  * Web browser: Firefox 72.0.2
  * Tor status: always
  * Opened port: no
  

### Step 2: Describe the problem:

In ZeroUp,KopyKate, and files menu i deleted some files.In the site directory still exit the picemap.msg and the deleted files too. This takes up space unnecessarily. The program does not want to delete them.Too much trash file collect this way the program.

#### Steps to reproduce:

  1. Upload a file to ZeroUp,KopyKate, etc.
  2. Delete file from the site and files menu too.
  
#### Observed Results:

In the site directory/user folder the picemap.msg and files does not deleted!

#### Expected Results:

Too much trash file collect this way the program.Which cannot be separated from the seeded files if the user seed for many files.

**Comment by HelloZeroNet** [comment]
It's normal if the .piecemap file does not get deleted as it's a separate optional file (we can change this behavior)

I just tested and it's works for me, please try to do the following:
 - Visit http://127.0.0.1:43110/Blog.ZeroNetwork.bit/?Post:126:New+version:+0.6.2, play the video
 - Go to http://127.0.0.1:43110/1HeLLo4uzjaLetFx6NH3PMwFP3qbRbTf3D/?Files
 - Filter to `local_peer` and try to delete the mp4 / piecemap file and see if it's got deleted from the directory.

**Comment by mx5kevin** [comment]
In ZeroBlog the picemap.msg and .mp4 file deleteion was worked perfectly. In ZeroUp i uploaded some files it doesn't work with them. The files are deleted from the cilent in the list. But still exit in the site/data/users/current user directory. The same with KopyKate Big! Each file was my own.

**Comment by styromaniac** [comment]
Yes, I've dealt with this issue, which makes it impossible to update an optional file when the old version is there. The old version remains.

I've seen KopyKate unable to delete videos, so the thought to be deleted videos clog up the site for seedboxes.
